format/format.go

```diff
@@ -999,7 +1028,7 @@ func (w *writer) objectWriter() entryWriter {
 	}
 }
 
-func (w *writer) listWriter() entryWriter {
+func (w *writer) listWriter(errs *ast.Errors) entryWriter {
```

Comment:
Is there some tricky scoping reason behind this, so we cannot use `w.errs` (as in other places) with `w.listWriter` and `w.ObjectWriter`?

Reply:
You are right, they are not necessary. I have removed them in the last commit